src/analytics/formTracker/formTracker.ts

```diff
@@ -10,84 +10,91 @@ export class FormTracker extends BaseTracker {
     super();
   }
 
+  private selectedFields: FormField[] = [];
+
+  private isExcludedType(element: HTMLInputElement): boolean {
```

Comment:
you need to add Unit tests for all your new functions.

Reply:
Ok will do